oasis/deployments.md

```diff
@@ -196,14 +196,60 @@ To isolate your deployments and increase security, you can use the private endpo
 Follow the steps outlined below to get started.
 
 {% hint 'info' %}
-The private endpoint feature is only available on the enterprise tier for
-deployments running on Microsoft Azure and Amazon Web Services (AWS).
+The private endpoint feature is only available on the enterprise tier of ArangoDB Cloud.
 {% endhint %}
 
 {% hint 'tip' %}
 Private endpoints on Microsoft Azure can be cross region; in AWS they should be located in the same region.
```

Comment:
What about GCP?
And what does "should" try to convey?

Reply:
This PR is about adding the GCP, so now all providers support it.